### PR TITLE
Adding new settings and fixing bugs for mini

### PR DESCRIPTION
**Pull request by LezaiYa**
1.Adding new settings for mini
_But it doesn't work. Help me._
2.Fix the bug that mini player's victory is not shown on the settlement page
_This bug comes from Tommy_

### CODE REVIEW

**Comment by NikoCat233** [comment]
Dont merge it. We are solving bugs

**Comment by NikoCat233** [comment]
Review pls

**Comment by NikoCat233** [comment]
1.Add setting: count grow up time in meeting
2.Improve evil mini kill cd logic

File: Patches/ExilePatch.cs [hunk 1]
```diff
@@ -170,33 +170,27 @@ static void WrapUpPostfix(GameData.PlayerInfo exiled)
         if (Witch.IsEnable) 
             Witch.RemoveSpelledPlayer();
 
-        if (HexMaster.IsEnable)
-            HexMaster.RemoveHexedPlayer();
-
```
Comment (Tommy-XL) [inline Patches/ExilePatch.cs:175]: Seems `HexMaster.RemoveHexedPlayer();` was removed

File: Patches/ExilePatch.cs [hunk 2]
```diff
@@ -170,33 +170,27 @@ static void WrapUpPostfix(GameData.PlayerInfo exiled)
...
+                if (swapper.Is(CustomRoles.Swapper))
+                {
+                    Swapper.Swappermax[swapper.PlayerId]--;
+                    Swapper.Vote.Clear();
+                    Swapper.VoteTwo.Clear();
+                    Main.SwapSend = false;
+                }
+            }
+        }
+
```
Comment (Tommy-XL) [inline Patches/ExilePatch.cs:186]: Is the code below necessary then?
image

Comment (NikoCat233) [reply Patches/ExilePatch.cs:186]: WTF does this code do.

File: Patches/ExilePatch.cs (hunk 1 above)
Comment (NikoCat233) [reply Patches/ExilePatch.cs:175]: I thought you removed it in your pr...

Comment (Tommy-XL) [reply Patches/ExilePatch.cs:175]: No

image

Comment (NikoCat233) [reply Patches/ExilePatch.cs:175]: Pls revert it
I dont have permission to directly write into lezaiya 's fork

**Review by Tommy-XL — APPROVED** [review]

**Pull request merged**